tests/particle_distribution_statistics/screen-output

```diff
@@ -0,0 +1,123 @@
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+Number of active cells: 80 (on 2 levels)
+Number of degrees of freedom: 5,314 (3,075+189+1,025+1,025)
+
+*** Timestep 0:  t=0 seconds, dt=0 seconds
+   Solving temperature system... 0 iterations.
+   Solving C_1 system ... 0 iterations.
+
+Number of active cells: 640 (on 3 levels)
+Number of degrees of freedom: 33,830 (19,683+1,025+6,561+6,561)
+
+*** Timestep 0:  t=0 seconds, dt=0 seconds
+   Solving temperature system... 0 iterations.
+   Solving C_1 system ... 0 iterations.
+
+Number of active cells: 1,536 (on 4 levels)
+Number of degrees of freedom: 76,062 (44,331+2,177+14,777+14,777)
+
+*** Timestep 0:  t=0 seconds, dt=0 seconds
+   Solving temperature system... 0 iterations.
+   Advecting particles...  done.
+   Solving C_1 system ... 0 iterations.
+
+   Postprocessing:
+     RMS, max velocity:                                                                 0.1 m/s, 0.1 m/s
+     Temperature min/avg/max:                                                           0 K, 0.8557 K, 1.266 K
+     Heat fluxes through boundary parts:                                                0.0005881 W, 0.0005881 W, 0.02101 W, -0.02296 W, 0.0005881 W, 0.0005881 W
+     Writing graphical output:                                                          output/solution/solution-00000
+     Writing particle output:                                                           output/particles/particles-00000
+     Particle Distribution Stats (stddev min/mean/max, mean min/max, absolute min/max): 0.0119338/0.0468226/0.0891903, 0.0053304/0.131836, 0/0.25
```

Comment:
Is there a way to understand these numbers analytically? I.e. is this what we expect if we create 4x4 particles per cell and use a uniform kernel function with granularity 2 and bandwidth 0.3? If there is no way to understand it in this context, let's talk if we can find a better test.